packages/nodes-base/nodes/Google/MyBusiness/PostDescription.ts

```diff
@@ -0,0 +1,1083 @@
+import {
+	NodeApiError,
+	NodeOperationError,
+	type IDataObject,
+	type IExecuteSingleFunctions,
+	type IN8nHttpFullResponse,
+	type INodeExecutionData,
+	type INodeProperties,
+	type JsonObject,
+} from 'n8n-workflow';
+
+import { addUpdateMaskPresend, handleDatesPresend, handlePagination } from './GenericFunctions';
+
+export const postOperations: INodeProperties[] = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		default: 'create',
+		noDataExpression: true,
+		displayOptions: { show: { resource: ['post'] } },
+		options: [
+			{
+				name: 'Create',
+				value: 'create',
+				action: 'Create post',
+				description: 'Create a new post on Google My Business',
+				routing: {
+					send: { preSend: [handleDatesPresend] },
+					request: {
+						method: 'POST',
+						url: '=/{{$parameter["account"]}}/{{$parameter["location"]}}/localPosts',
+					},
+				},
+			},
+			{
+				name: 'Delete',
+				value: 'delete',
+				action: 'Delete post',
+				description: 'Delete an existing post',
+				routing: {
+					request: {
+						method: 'DELETE',
+						url: '=/{{$parameter["post"]}}',
+						ignoreHttpStatusErrors: true,
+					},
+					output: {
+						postReceive: [
+							async function (
+								this: IExecuteSingleFunctions,
```

Comment:
Can you extract this to GenericFunctions?

Reply:
I have extracted it.